### ISSUE TITLE
[CovidQA Experiment] ModuleNotFoundError: No module named 'pyserini.analysis.pyanalysis'

### ISSUE THREAD

**Issue by justinborromeo** [castorini/pygaggle#70]
I'm trying to replicate the CovidQA experiment and am receiving the following error when running the following command:

```
$ python -um pygaggle.run.evaluate_kaggle_highlighter --method random 
\
>                                                     --dataset data/kaggle-lit-review-0.2.json \
>                                                     --index-dir indexes/lucene-index-cord19-paragraph-2020-05-12
Traceback (most recent call last):
  File "/home/justin/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/justin/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/justin/Justin/School/data-systems-group/pygaggle/pygaggle/run/evaluate_kaggle_highlighter.py", line 16, in <module>
    from pygaggle.rerank.bm25 import Bm25Reranker
  File "/home/justin/Justin/School/data-systems-group/pygaggle/pygaggle/rerank/bm25.py", line 6, in <module>
    from pyserini.analysis.pyanalysis import get_lucene_analyzer, Analyzer
ModuleNotFoundError: No module named 'pyserini.analysis.pyanalysis'
```

I'm using Anserini 0.9.5, Pyserini 0.9.4, and PyTorch 1.4.0.  Is there something I'm missing?

Also, the same error occurs when reranking with MonoT5 from the MSMarco-Document experiment.

**Comment by lintool** [comment]
We refactored the module structure in pyserini: https://github.com/castorini/pyserini/blob/master/docs/release-notes/release-notes-v0.9.3.1.md

`pyserini.analysis.pyanalysis`-> https://github.com/castorini/pyserini/tree/master/pyserini/analysis

Can you send PR to fix please?

**Comment by ronakice** [comment]
Resolved by #71
